Sources/Pokepay/Pokepay.swift

```diff
@@ -231,17 +235,17 @@ public struct Pokepay {
             }
         }
 
-        public func scanToken(_ token: String, amount: Double? = nil, accountId: String? = nil, products: [Product]? = nil,couponId: String? = nil, strategy: TransactionStrategy? = .pointPreferred,
+        public func scanToken(_ token: String, amount: Double? = nil, accountId: String? = nil, products: [Product]? = nil, couponId: String? = nil, strategy: TransactionStrategy? = .pointPreferred,
                               handler: @escaping (Result<UserTransaction, PokepayError>) -> Void = { _ in }) {
-            if token.hasPrefix("\(wwwBaseURL)/cashtrays/") {
+            if token.hasPrefix("\(wwwBaseURL)/cashtrays/") || token.hasPrefix("\(defaultWwwBaseURL)/cashtrays/") {
                 let uuid = String(token.suffix(token.utf8.count - "\(wwwBaseURL)/cashtrays/".utf8.count))
```

Comment:
(REQUEST) I think we need to fix this line as well.
ex.
wwwBaseURL => 'https://www.univcoop.jp'
qr_string => 'https://www.pokepay.jp/cashtrays/xxxxxxxxxxxxxxxx'
uuid wont be 'xxxxxxxxxxxxxxxx'

Reply:
Yes, that's correct.  Thank you for catching that.  I'll fix it now.